config/jobs/kubernetes-sigs/service-apis/service-apis-config.yaml

```
@@ -23,3 +23,26 @@ presubmits:
         # docker-in-docker needs privileged mode.
         securityContext:
           privileged: true
+  - name: pull-service-apis-test
+    annotations:
+      testgrid-dashboards: sig-network-service-apis
+      testgrid-tab-name: test
+    labels:
+      preset-kind-volume-mounts: "true"
+      preset-dind-enabled: "true"
+    decorate: true
+    path_alias: sigs.k8s.io/service-apis
+    always_run: true
+    skip_report: false
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20201215-73fe430-master
+          command:
+            # generic runner script, handles DIND, bazelrc for caching, etc.
+            - runner.sh
+          args:
+            - make
+            - test
+          # docker-in-docker needs privileged mode.
+          securityContext:
+            privileged: true
```

```suggestion
            privileged: true
            
```
Lint failing on new line

---

Think I fixed it now, nice catch :)